packages/custom_lint/lib/src/analyzer_plugin/plugin_link.dart

```diff
@@ -188,6 +212,12 @@ final pluginMetasForContextRootProvider = Provider.autoDispose
 
 // TODO extract magic value
       if (dependencyPubspec.hasDependency('custom_lint_builder')) {
+        ref
```

Comment:
This slipped my review, but that doesn't work. Providers aren't allowed to modify other providers during their init